src/core/server/decorators/interaction.ts

```diff
@@ -0,0 +1,37 @@
+import alt from 'alt-server';
+import { Interaction } from '../../shared/interfaces/interaction';
+import { Athena } from '../api/athena';
+/**
+ * Decorator for Athena's Interaction Controller
```

Comment:
A little space here.

```suggestion
import { Athena } from '../api/athena';

/**
 * Decorator for Athena's Interaction Controller
```